.idea/fileTemplates/includes/C File Header.h

```diff
@@ -0,0 +1,28 @@
+#if ($HEADER_COMMENTS)
```

Comment:
I think this change is coming from another PR, but it would be a good idea to avoid file names containing spaces if possible.